bridge/Bridge.h

```diff
@@ -43,6 +43,8 @@ class Engine;
 class MixerManager;
 class ApiRequestHandler;
 
+std::vector<transport::SocketAddress> gatherInterfaces(const config::Config& config);
```

Comment:
Perhaps it make sense to move this other place.

Reply:
Moved to new files utils\GatherInterfaces.h/.cpp